main.cpp

```diff
@@ -35,89 +35,109 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
 
 /*
  1)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun: missile
+ action 1: the missile is fired
+ action 2: the missile locks onto a target
+ action 3: the missile explodes
  */
-
+missile.fire();
+missile.selectTarget();
+missile.explode
```

Comment:
`is fired`  Try to not use `is` in your actions.  `is` is often used to query object state.   
an improvement would be `the missile launches` 
```
missile.launch();
```
"The missile locks onto a target" means something different from `selectTarget();`   
Selecting the target is what you do before you lock onto it. 
`missile.explode` is missing the `();`

- [ ] revise action 1 and revise pseudo code for action 1
- [ ] revise pseudo code for action 2.
- [ ] add missing ();